examples/MicroserviceExample/Utils/Messaging/MessageReceiver.cs

```diff
@@ -58,7 +58,7 @@ public void StartConsumer()
         public void ReceiveMessage(BasicDeliverEventArgs ea)
         {
             // Extract the PropagationContext of the upstream parent from the message headers.
-            var parentContext = Propagator.Extract(default, ea.BasicProperties, this.ExtractTraceContextFromBasicProperties);
+            var parentContext = Propagators.DefaultTextMapPropagator.Extract(default, ea.BasicProperties, this.ExtractTraceContextFromBasicProperties);
```

Comment:
On the first message received, the `DefaultTextMapPropagator` is the noop instance. By the second message received it has been initialized to the composite propagator that extracts trace context and baggage.

Reply:
can we "touch" SDK somewhere super early......

https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs#L35-L40

Reply:
Resolved this by touching the `Sdk` class in OpenTelemetry.Extensions.Hosting.

Reply:
A gentle touch, and SDK behaves!